trove/derive/__init__.py

```diff
@@ -1,19 +1,17 @@
 from . import (
     sharev2_elastic,
-    osfmap_json,
+    osfmap_json_mini,
     oaidc_xml,
 )
 
-
 DERIVER_SET = (
     sharev2_elastic.ShareV2ElasticDeriver,
-    osfmap_json.OsfmapJsonDeriver,
+    osfmap_json_mini.IndexcardJsonDeriver,
```

Comment:
hmm, want to reduce potential confusion with the "mini" fully replacing the original `osfmap_json`...

short of redesigning `trove.derive` to be less confusing, tho, should make sure the two have distinct `deriver_iri` -- could copy the [old deriver_iri](https://github.com/CenterForOpenScience/SHARE/blob/077710829626ea053af3b81d700b54f2f8e7fdcb/trove/derive/osfmap_json.py#L16-L19) to the new "mini" class, then update the old to return `TROVE['derive/osfmap_json_full']` instead (this way current api will start using the "mini", but the "full" has a distinct identifier in case it's ever used another way)